app/models/piece.rb

```diff
@@ -4,6 +4,7 @@ class Piece < ApplicationRecord
   def move!(x, y)
     return false unless valid_move?(x, y)
     update_attributes(x_position: new_x, y_position: new_y)
+    Move.create(piece_id: id)
```

Comment:
Well, it's called when moving a piece, and since we haven't attempted that request yet, there's no error, but as soon as somebody does, it would be an issue without the Move model actually being defined.